bin/zephir

```diff
@@ -19,9 +19,8 @@ if [ -z "$ZEPHIRDIR" ]; then
   fi
 fi
 
-if [[ $1 && $2 && $3 && "$1"=="-c" ]]; then
+if [ $1 -a $2 -a $3 -a $1 = "-c" ]; then
```

Comment:
I don't think /bin/sh matters here. What matters is I changed it to `[` which is the same as executing `test`. Here's the documentation for **test** on FreeBSD.
https://www.freebsd.org/cgi/man.cgi?test

I would think it work the same on any system because I see the mention of POSIX, but then I noticed this sentence...
"In particular, only expressions containing -a, -o, ( or ) can be ambiguous."

I can check this on my arch linux system at home. I wonder if this is what I should be checking as well.
https://linux.die.net/man/1/test